packages/core/useScrollLock/index.ts

```diff
@@ -63,11 +64,13 @@ export function useScrollLock(
     const ele = (unref(element) as HTMLElement)
     if (!ele || !isLocked.value)
       return
-    isIOS && touchMoveListener?.()
+    isIOS && stopTouchMoveListener?.()
     ele.style.overflow = initialOverflow
     isLocked.value = false
   }
 
+  tryOnScopeDispose(unlock)
```

Comment:
I'm not sure about this.
Maybe we just need to recycle events?

```ts
tryOnScopeDispose(() => {
  stopTouchMoveListener?.()
})
```

Reply:
useEventListener already disposes automatically and we don't need this.

Reply:
Unfortunately, the listener is started within the `lock`, should be in the global scope